test_util/helpers.py

```diff
@@ -23,81 +24,139 @@ def path_join(p1, p2):
 
 class DcosUser:
     """A lightweight user representation."""
-    def __init__(self, auth_json):
+    def __init__(self, auth_json: dict):
```

Comment:
Well, the ambiguity of `auth` is more worrisome to me than the `_json` suffix. So, I'd prefer `credentials`. If you then prefer `credentials_json`, well -- ok.